docs/docs-beta/docs/tutorials/category-two/prompt-engineering/prompts.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Prompts
+description: Generating the first prompt
+last_update:
+  author: Dennis Hume
+sidebar_position: 20
+---
+
+We will be working with the API for the National Renewable Energy Laboratory ([NREL](https://www.nrel.gov/)). This API allows us to find nearby alternative fuel stations. To use this API we need to provide our latitude and longitude and some information about our vehicle. Trying to make this user friendly, we might want to put an AI in front to translate free text into the information we need.
```

Comment:
```suggestion
We will be working with the API for the National Renewable Energy Laboratory ([NREL](https://www.nrel.gov/)). This API allows us to find nearby alternative fuel stations. To use this API, we need to provide our latitude and longitude and some information about our vehicle. Trying to make this user friendly, we might want to put an AI in front to translate free text into the information we need.
```

Reply:
You may want to lead with something like "We're building an app to help users find nearby alternative fuel stations. To that end, we will be working with the API for the National Renewable Energy Laboratory, which requires latitude and longitude information, and some vehicle information" (feel free to adapt!)